curriculum-ui/src/App.tsx

```diff
@@ -1,22 +1,24 @@
 import React from 'react';
+import 'bootstrap/dist/css/bootstrap.min.css'; 
 import './App.css';
-import 'bootstrap/dist/css/bootstrap.min.css';
-import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
 import { store } from './Store';
 import { Provider } from 'react-redux';
-import CreateCurriculumFormComponent from './components/create-curriculum-form-component/CreateCurriculumFormContainer';
+import { CreateSkillComponent } from './components/create-skill-component/CreateSkillComponent';
+import  ViewAllSkillsComponent  from './components/view-all-skill-component/ViewAllSkillsComponent';
+
+
 function App() {
   return (
     <Provider store = {store}>    
-       {/* Remove once App.test.tsx is fixed*/}
-      <div className="App">
-        <Router> 
-          {/* Navbar */}
-          <Switch> 
-            <Route path="/curriculum/create" component={CreateCurriculumFormComponent}/>
-          </Switch>
-        </Router>          
-      </div>
+    <div className="App">
+      <Router> 
+        <Switch>
+          <Route path ='/CreateSkill' component = {CreateSkillComponent}/>
+          <Route path ='/ViewAllSkills' component = {ViewAllSkillsComponent}/>           
+        </Switch>
```

Comment:
you should probably have the other route in dev as well, because otherwise you are getting rid of someone elses code